release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1741,7 +1745,7 @@ In the following tables, features are marked with the following statuses:
 
 |Network Observability Operator
 |Not Available
-|Not Available
+|General Availability
```

Comment:
@skrthomas can you explain this change? I looked at the preview but couldn't figure it out myself  :sweat_smile:

Reply:
@skrthomas - It looks like 2nd column is for 4.11 which should be `General Availability` as well?

Reply:
This is really confusing. I guess, I'm going off the language [here](https://access.redhat.com/support/policy/updates/openshift#noo):
**Commencing with OpenShift 4.12 release**, network observability is provided and available as an optional, installable component for OpenShift 4.10 and later. Network Observability is shipped inclusive as part of the OpenShift Container Platform subscription

That's where I'm getting the, we have GA in 4.12, and it is provided as a component for OCP 4.10, 4,11. I'm not sure if I'm thinking about this correctly or not. I guess @mcurry-rh can maybe weigh in on this. Should we be listing Network Observability as GA in 4.11 too?